zcutil/fetch-params.sh

```diff
@@ -225,7 +236,7 @@ EOF
     fetch_params "$SAPLING_SPROUT_GROTH16_NAME" "$PARAMS_DIR/$SAPLING_SPROUT_GROTH16_NAME" "b685d700c60328498fbde589c8c7c484c722b788b265b72af448a5bf0ee55b50"
```

Comment:
Consider changing the Sprout part to
```shell
# Delete Sprout parameters:
delete_params "$PARAMS_DIR/$SPROUT_PKEY_NAME" "8bc20a7f013b2b58970cddd2e7ea028975c88ae7ceb9259a5344a16bc2c0eef7"
delete_params "$PARAMS_DIR/$SPROUT_VKEY_NAME" "4bd498dae0aacfd8e98dc306338d017d9c08dd0918ead18172bd0aec2fc5df82"
```
where
```shell
delete_params() {
    # We only set these variables inside this function,
    # and unset them at the end of the function.
    output="$1"
    expectedhash="$2"
    if [ -f "$output" ]
    then
        "$SHA256CMD" $SHA256ARGS -c <<EOF
$expectedhash  $output
EOF
        # Check the exit code of the shasum command:
        CHECKSUM_RESULT=$?
        if [ $CHECKSUM_RESULT -eq 0 ]; then
            rm -f "$output"
        else
            echo "Checksum of old Sprout parameter file ${output} is not as expected -- not deleting!" >&2
            exit 1
        fi
    fi
    unset -v output
    unset -v expectedhash
}
```

Reply:
I'll file a separate issue for this.

Reply:
#6295